### ISSUE TITLE
I can't build the solution

### ISSUE THREAD

**Issue by sebagomez** [tunnelvisionlabs/LangSvcV2#100]
I'm fixing to add some features to the StringTemplate coloring extension so I forked and cloned the repo but I can't build it.

I had to update every project so it won't try to sign the assembly, but now I'm having this error message:

`Severity	Code	Description	Project	File	Line	Suppression State
Error		Could not add the file "C:\code\sebastian\LangSvcV2\packages\Tvl.VisualStudio.OutputWindow.3.0.0\lib\net40\Tvl.VisualStudio.OutputWindow.vsix" to the zip package "bin\Debug\Tvl.VisualStudio.Framework.vsix". Could not find a part of the path 'C:\code\sebastian\LangSvcV2\packages\Tvl.VisualStudio.OutputWindow.3.0.0\lib\net40\Tvl.VisualStudio.OutputWindow.vsix'.	Tvl.VisualStudio.Framework	C:\Users\user\.nuget\packages\microsoft.vssdk.buildtools\15.1.192\tools\VSSDK\Microsoft.VsSDK.targets	600	
`

Any help would be greatly appreciated

**Comment by nolisj** [comment]
@sebagomez , I also had the same problem. To fix that error message, what I did was I created a new (and separate) throwaway Extensibility project and reference the Tvl.VisualStudio.OutputWindow (using "Install-Package Tvl.VisualStudio.OutputWindow -Version 3.0.0" command in my Package Manager Console). Once this temporary project was successfully built, I copied the generated folder under the packages folder from this solution, then pasted them to my LangSvcV2\packages solution folder. (FYI, I also manually created this packages folder since it was not automatically created when I build my LangSvV2 solution.) After that, the error was gone.

I hope this helps.
